core/assets/jsons/global.json

```diff
@@ -0,0 +1,68 @@
+{
+  "player":  {
+    "radius":      	0.4,
+    "bodytype":    	"dynamic",
+    "density":        1.0,
+    "friction":       0.0,
+    "restitution":    0.0,
+    "force": 		 80.0,
+    "damping": 		 10.0,
+    "maxspeed": 	  5.0,
+    "startframe":       0,
+    "walklimit":		4,
```

Comment:
What is the walk limit?

Reply:
This is the cooldown limit between walk animations